tutorials/boundary_conditions.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #47.            values = np.ones((self.nd, bg.num_cells))
</pre><p>I think in general it's safer to initialize with zeros.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>